rskj-core/src/main/java/co/rsk/config/RskSystemProperties.java

```diff
@@ -482,6 +483,17 @@ public double getTopBest() {
         return value;
     }
 
+    public GasCalculator.GasCalculatorType getGasCalculatorType() {
+        String value = configFromFiles.getString("miner.gasPriceCalculatorType");
```

Comment:
could you extract the key string to constants?

Reply:
oh sure, good point